pkg/policy/k8s/network_policy.go

```diff
@@ -31,22 +31,16 @@ func (p *policyWatcher) addK8sNetworkPolicyV1(k8sNP *slim_networkingv1.NetworkPo
 	}
 	scopedLog = scopedLog.WithField(logfields.K8sNetworkPolicyName, k8sNP.ObjectMeta.Name)
 
-	opts := policy.AddOptions{
+	opts := &policytypes.PolicyUpdate{
```

Comment:
Symbol `opts` is a bit misleading here, maybe call `UpdatePolicy` here like elsewhere?